src/components/MapDetail/MapDetail.js

```diff
@@ -0,0 +1,102 @@
+import React from 'react'
+import {
+  Box,
+  Button,
+  CheckBox,
+  Heading,
+  Text
+} from 'grommet'
+import styled from 'styled-components'
+import { Close } from 'grommet-icons'
+import FalklandsMap from 'images/falk_map.png'
+import AssociatedSubjects from './components/AssociatedSubjects'
+import Charts from './components/Charts'
+import Timeline from './components/Timeline'
+import ResponsiveImage from 'components/ResponsiveImage'
+
+const StyledHeading = styled(Heading)`
+  font-family: Neuton;
+`
+
+const StyledText = styled(Text)`
+  font-family: Neuton;
+`
+
+const HeadingTwo = styled(StyledHeading)`
+  font-size: 2.5rem;
+  font-weight: 300;
+`
+
+const Uppercase = styled(Text)`
+  text-transform: uppercase;
+`
+
+export default function MapDetail() {
+  return (
+    <Box
+      background='sand'
+      border={{ color: 'kelp' }}
+      fill
+      gap='medium'
+      pad={{ horizontal: 'large', vertical: 'xsmall' }}
+      width={{ max: '70rem' }}
+    >
+      <Box
+        border={{ color: 'kelp', side: 'bottom' }}
+        direction='row'
+        justify='between'
+        pad={{ vertical: 'small' }}
+      >
+        <StyledHeading
+          color='kelp'
+          level='4'
+          margin='none'
+        >
+          Map Detail
+        </StyledHeading>
+        <Button
+          alignSelf='end'
+          color='kelp'
+          gap='xsmall'
+          icon={<Close color='black' size='small' />}
+          label={<Uppercase size='xsmall'>Close</Uppercase>}
+          plain
+          reverse
+        />
+      </Box>
+
+      <Box direction='row' gap='large'>
+        <Box gap='xsmall'>
+          <Box gap='xsmall'>
+            <HeadingTwo
+              color='kelp'
+              level='2'
+              margin='none'
+            >
+              Falkland Islands
+            </HeadingTwo>
+            <Box align='center' direction='row' justify='between'>
+              <Box direction='row' gap='small'>
```

Comment:
I think left-alignment and the letter-spacing might be off from InVision

Reply:
Good catch. I didn't account for letter spacing.